firefox-ios/firefox-ios-tests/Tests/XCUITests/InactiveTabsTest.swift

```diff
@@ -0,0 +1,134 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+final class InactiveTabsTest: BaseTestCase {
+    override func setUp() {
+        // Load 20 tabs. 19 tabs are inactive.
+        let tabsDatabase = "tabsState20.archive"
+        launchArguments = [
+            LaunchArguments.PerformanceTest,
+            LaunchArguments.SkipIntro,
+            LaunchArguments.SkipWhatsNew,
+            LaunchArguments.SkipETPCoverSheet,
+            LaunchArguments.SkipContextualHints,
+            LaunchArguments.DisableAnimations,
+        ]
+        launchArguments.append(LaunchArguments.LoadTabsStateArchive + tabsDatabase)
```

Comment:
You need to add the test there not the tabsDatabase...
launchArguments.append(LaunchArguments.LoadTabsStateArchive + fixtures[functionName]!)

[https://github.com/mozilla-mobile/firefox-ios/blob/739e89d86df17511c7382ed0790333c[…]ox-ios/firefox-ios-tests/Tests/XCUITests/PerformanceTests.swift](https://github.com/mozilla-mobile/firefox-ios/blob/739e89d86df17511c7382ed0790333cd97f43f84/firefox-ios/firefox-ios-tests/Tests/XCUITests/PerformanceTests.swift#L38)